css/styles.css

```diff
@@ -0,0 +1,142 @@
+body {
```

Comment:
please use scss and put the colors and some of the main sizes into the constants